pyomo/contrib/pynumero/examples/structured/tests/test_nlp_transformations.py

```diff
@@ -206,24 +206,24 @@ def setUpClass(cls):
 
```

Comment:
This file will disappear with #1936 . I'm surprised these changes needed to happen since these tests do not run or work.